R/rope.R

```diff
@@ -43,13 +43,20 @@ as.double.rope <- function(x, ...) {
 
 #' @export
 print.rope <- function(x, ...) {
-  cat(sprintf(
-    "%.2f%% of the %s%% CI is in ROPE [%.2f, %.2f]",
-    x$ROPE_Percentage,
-    x$CI,
-    x$ROPE_low,
-    x$ROPE_high
-  ))
+  if (is.data.frame(x)) {
+    cat(sprintf("# Proportions of samples inside the ROPE [%.2f, %.2f]\n\n", x$ROPE_low[1], x$ROPE_high[1]))
```

Comment:
I think it might be worth to specify to what "samples" we refer to, i.e., "samples from the credible interval" or "samples from the posterior distribution" (in the case ci = 1). And indicate the percentage of CI even when there is no multiple CIs